cypress/e2e/hub/signature-keys.cy.ts

```diff
@@ -2,16 +2,59 @@ import { SignatureKeys } from './constants';
 import { pulpAPI } from '../../support/formatApiPathForHub';
 
 describe('Signature Keys', () => {
-  before(() => {
+  beforeEach(() => {
     cy.hubLogin();
   });
 
   it('should render the signature keys page', () => {
```

Comment:
The test `should display empty state when no content when no data` could be combined here.